v2/emailpassword/common-customizations/sessions/multiple-api-endpoints.mdx

```diff
@@ -28,9 +28,9 @@ To enable use of sessions for multiple API endpoints, you need to use the `sessi
 - `sessionTokenBackendDomain` in the frontend config must match the `cookieDomain` set in the backend config.
```

Comment:
not related to this file, but, i think there is a section in the docs that talks about session error handlers and how to configure them. If such a section exists,. we need to modify it to also talk about the new type of error for duplicate cookie

Reply:
done